apps/aens/src/aens_utils.erl

```diff
@@ -71,9 +79,9 @@ check_name_owner(Name, AccountPubKey) ->
     end.
 
 check_claimed_status(Name) ->
```

Comment:
The name of this function is slightly strange? `is_claimed`?

Reply:
I wanted to follow `check_` prefix for all the checks. Or maybe you want to replace `is_revoked` with `is_claimed` (the first was slightly better in the other place, that's why I used it) 🤔 ?

But I'm not devoted to any of these, and can adjust if you prefer ☺️

Reply:
No strong preference - just found it slightly strange...